src/core/Form/Combobox/ComboboxEmptyItem/ComboboxEmptyItem.baseStyles.tsx

```diff
@@ -0,0 +1,18 @@
+import { css } from 'styled-components';
+import { defaultThemeTokens as theme } from '../../../theme';
+import { font } from '../../../theme/reset';
+
+export const baseStyles = css`
+  &.fi-combobox-empty-item {
+    ${font(theme)('actionElementInnerText')}
+    padding: 10px;
```

Comment:
Should we use a token here or is this "component internals"?

Reply:
Good find. I think we could use value from token here with the `padding`.

Reply:
Changed `padding` to now use the token: 
``` css
padding: ${theme.spacing.insetL};
```